ci/workflows/main-cron.yml

```diff
@@ -119,17 +119,22 @@ steps:
     retry: *auto-retry
 
   - label: "fuzz test"
-    command: "ci/scripts/fuzz-test.sh"
+    command: "ci/scripts/cron-fuzz-test.sh -p ci-release"
+    depends_on:
+      - "build"
     plugins:
       - ./ci/plugins/swapfile
       - gencer/cache#v2.4.10: *cargo-cache
       - docker-compose#v3.9.0:
           run: rw-build-env
           config: ci/docker-compose.yml
+          mount-buildkite-agent: true
           environment:
             - RUN_SQLSMITH: 1
+            - SQLSMITH_COUNT: 1000
             - RW_RANDOM_SEED_SQLSMITH: true
-    timeout_in_minutes: 10
+      - ./ci/plugins/upload-failure-logs
+    timeout_in_minutes: 20
```

Comment:
Note to reviewer: Increased since it's running e2e fuzz as well now. Additionally, it's running for 1000 queries, instead of 100 in `main.yml` and `pull-request.yml`, so more time is needed. 

From looking at recent `main-cron`, this time seems to be sufficient. May adjust again if needed.